shared/src/store/action-reducers/utils/log.ts

```diff
@@ -64,6 +66,86 @@ export function logRadiogram(
         )
     );
 }
+export function logTreatmentStatusChangedInSimulatedRegion(
+    state: Mutable<ExerciseState>,
+    treatmentProgress: TreatmentProgress,
+    simulatedRegionId: UUID,
+    additionalTags: Tag[] = []
+) {
+    if (!logActive(state)) return;
+
+    const simulatedRegion = getElement(
+        state,
+        'simulatedRegion',
+        simulatedRegionId
+    );
+
+    state.logEntries!.push(
+        new LogEntry(
+            `Der Behandlungszustand im simulierten Bereich ${simulatedRegion.name} ist zu ${treatmentProgressToGermanNameDictionary[treatmentProgress]} gewechselt.`,
+            [
+                ...additionalTags,
+                createTreatmentProgressTag(state, treatmentProgress),
+                createSimulatedRegionTag(state, simulatedRegionId),
+            ],
+            state.currentTime
+        )
+    );
+}
+
+export function logLastPatientTransportedInSimulatedRegion(
+    state: Mutable<ExerciseState>,
+    patientStatus: PatientStatus,
+    simulatedRegionId: UUID,
+    additionalTags: Tag[] = []
+) {
+    if (!logActive(state)) return;
+
+    const simulatedRegion = getElement(
+        state,
+        'simulatedRegion',
+        simulatedRegionId
+    );
+
+    state.logEntries!.push(
+        new LogEntry(
+            `Der letzte Patient der Kategorie ${statusNames[patientStatus]} im simulierten Bereich ${simulatedRegion.name} ist abtransportiert worden`,
+            [
+                ...additionalTags,
+                createPatientStatusTag(state, patientStatus),
+                createSimulatedRegionTag(state, simulatedRegionId),
+            ],
+            state.currentTime
+        )
+    );
+}
+
+export function logLastPatientTransportedInMultipleSimulatedRegions(
+    state: Mutable<ExerciseState>,
+    patientStatus: PatientStatus,
+    managingSimulatedRegionId: UUID,
+    additionalTags: Tag[] = []
+) {
+    if (!logActive(state)) return;
+
+    const simulatedRegion = getElement(
+        state,
+        'simulatedRegion',
+        managingSimulatedRegionId
+    );
+
+    state.logEntries!.push(
+        new LogEntry(
+            `Der letzte Patient der Kategorie ${statusNames[patientStatus]} in allen Bereichen, die von der TO in ${simulatedRegion.name} verwaltet werden, ist abtransportiert worden`,
+            [
+                ...additionalTags,
+                createPatientStatusTag(state, patientStatus),
+                createSimulatedRegionTag(state, managingSimulatedRegionId),
+            ],
+            state.currentTime
+        )
+    );
+}
```

Comment:
Please merge these two function into one `logPatientTransport` function, that gets passed the description as a string

Reply:
Done